net/smartdns/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2018-2019 Nick Peng (pymumu@gmail.com)
+# This is free software, licensed under the GNU General Public License v3.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=smartdns
+PKG_VERSION:=1.2019.28
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://www.github.com/pymumu/smartdns.git
+PKG_SOURCE_VERSION:=cc93156ee3af9de89e8e6db72554a3790170ce0b
+PKG_MIRROR_HASH:=89199413a3860d8245da6e4c6f171d449b89ec56162579004fa2166a29d20159
+
+PKG_MAINTAINER:=Nick Peng <pymumu@gmail.com>
+PKG_LICENSE:=GPL-3.0-or-later
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+MAKE_ARGS:=CFLAGS="$(TARGET_CFLAGS) -DSMARTDNS_VERION=\\\"$(PKG_VERSION)\\\"" CC=$(TARGET_CC) 
```

Comment:
Is this really needed? I would think

TARGET_CFLAGS += -DSMARTDNS_VERION=\\\"$(PKG_VERSION)\\\"

does the same thing.